distribution/docs/src/main/resources/content/_dependencyList/ddf-dependency-list.adoc

```diff
@@ -153,26 +153,26 @@
 * org.apache.cxf:cxf-rt-ws-security:jar:3.1.11
 * org.apache.felix:org.apache.felix.configadmin:jar:1.8.14
 * org.apache.felix:org.apache.felix.coordinator:jar:1.0.2
-* org.apache.felix:org.apache.felix.fileinstall:jar:3.6.0
-* org.apache.felix:org.apache.felix.framework:jar:5.6.6
+* org.apache.felix:org.apache.felix.fileinstall:jar:3.6.4
+* org.apache.felix:org.apache.felix.framework:jar:5.6.8
 * org.apache.felix:org.apache.felix.utils:jar:1.10.0
 * org.apache.ftpserver:ftplet-api:jar:1.0.6
 * org.apache.ftpserver:ftpserver-core:jar:1.0.6
 * org.apache.geronimo.specs:geronimo-servlet_3.0_spec:jar:1.0
 * org.apache.httpcomponents:httpclient:jar:4.5.3
 * org.apache.httpcomponents:httpcore:jar:4.4.6
 * org.apache.httpcomponents:httpmime:jar:4.5.3
-* org.apache.karaf.bundle:org.apache.karaf.bundle.core:jar:4.1.2
-* org.apache.karaf.features:org.apache.karaf.features.core:jar:4.1.2
-* org.apache.karaf.features:standard:xml:features:4.1.2
-* org.apache.karaf.jaas:org.apache.karaf.jaas.boot:jar:4.1.2
-* org.apache.karaf.jaas:org.apache.karaf.jaas.config:jar:4.1.2
-* org.apache.karaf.jaas:org.apache.karaf.jaas.modules:jar:4.1.2
-* org.apache.karaf.shell:org.apache.karaf.shell.console:jar:4.1.2
-* org.apache.karaf.shell:org.apache.karaf.shell.core:jar:4.1.2
-* org.apache.karaf:apache-karaf:tar.gz:4.1.2
-* org.apache.karaf:apache-karaf:zip:4.1.2
-* org.apache.karaf:org.apache.karaf.util:jar:4.1.2
+* org.apache.karaf.bundle:org.apache.karaf.bundle.core:jar:4.1.3
```

Comment:
❓ Shouldn't those be `4.1.4` instead of `4.1.3`?

Reply:
oops! Fixed